src/SFML/Graphics/Font.cpp

```diff
@@ -502,25 +474,6 @@ bool Font::isSmooth() const
 }
 
 
-////////////////////////////////////////////////////////////
-Font& Font::operator=(const Font& right)
-{
-    Font temp(right);
-
-    std::swap(m_fontHandles, temp.m_fontHandles);
-    std::swap(m_isSmooth, temp.m_isSmooth);
-    std::swap(m_info, temp.m_info);
-    std::swap(m_pages, temp.m_pages);
-    std::swap(m_pixelBuffer, temp.m_pixelBuffer);
-
-#ifdef SFML_SYSTEM_ANDROID
-    std::swap(m_stream, temp.m_stream);
-#endif
-
-    return *this;
-}
```

Comment:
I'm not sure why this was originally implemented as a swap but this PR would change it to a normal copy assignment. Is that prone to break anything?

Reply:
The code was just using the Copy-and-Swap Idiom.
Instead of duplicating all the exceptional code in both the copy constructor and the assignment operator, the assignment operator calls the copy constructor (by creating the temp object) and then swaps the result of the copy constructor into `this`.

So this was just to reuse the code in the copy constructor, it is functionally equivalent to a normal copy assignment.